src/lib/components/input/input.js

```diff
@@ -36,10 +37,10 @@ const Input = ({
     handleInputChange(event, name, handleValidations, onChangeFunction);
 
   return (
-    <div className={`${styles[`theme-${theme}`]} ${containerClassName}`}>
+    <div className={classNames(styles[`theme-${theme}`], containerClassName)}>
       <div className={styles.wrapper}>
         <input
-          className={`${className} ${isValidationOk() && styles.error}`}
+          className={`${className} ${isValidationOk() ? styles.error : ''}`}
```

Comment:
if you use classNames, you could do: `className={classNames(className, { [styles.error]: isValidationOk() })}`